Load msgs for selected room

ergunirem opened the pull request:
Backend: 
- addMessage -> now it fetches the room and created the message entity with room and user elements.
- getMessagesForRoom -> message service finds messages based on room name

Frontend: 
- I removed ScrollPanel prime component and used overflow: scroll in style part for all-messages div. That solved my problem with not being able to keep scrollbar at the bottom when a new message is sent.
- Adjusted all-messages div height to different screen size options.
- Added user info and date to msg cards
- Added key to route-view in chat-view which made the component to mount once the route changes
 
Please play with sending all kinds of messages to let me know about any problem that did not occur to me. For the style of the card, we can discuss things. Atm, I just wanted to create the PR.

TODO: Display own messages on the right and other msgs on the left, check codam computer white scrollbar issue

Mollie-S reviewed (commented):
The messages are saved in the rooms they belong to. Great.
Just a couple of things I've noticed. Probably sth to discuss for the future.
-  Styles:
   - Looks like the username and avatar have to be aligned to the left, not centered. 
   - I would also place the message time to the right from the username to save some space as it's not usually the most important information
<img width="1431" alt="Screenshot 2022-06-11 at 21 16 12" src="https://user-images.githubusercontent.com/61191631/173202017-14a8974b-a781-4600-9330-d7e6ac988deb.png">

- I've tried to copy and paste 1000 pages text. Visually the text has been placed into the input, then disappeared but was obviously not sent. The server-side logs:
```
[Nest] 95613  - 11/06/2022, 21:31:35     LOG [ChatGateway] Client disconnected
[Nest] 95613  - 11/06/2022, 21:31:36     LOG [ChatGateway] Client connected
```
That's totally ok, but as a user I got confused as was unsure whether the message has been sent or not :) 
I tried to send the same message in slack and got this:
<img width="518" alt="Screenshot 2022-06-11 at 21 17 33" src="https://user-images.githubusercontent.com/61191631/173202391-bef88c14-8ff7-4762-8fe6-fb0dc1f4cb34.png">
Not a priority though.

ergunirem commented:
Improvements based on the comments:
- added max-length to InputText: now it is 200 but we can change afterwards
- added max-width to card element so that it's not very wide with long texts
- added time as a chip next to username&useravatar chip & aligned them to sides

Now it looks like thi
<img width="1221" alt="Screenshot 2022-06-12 at 12 51 17" src="https://user-images.githubusercontent.com/57744064/173229768-af70f3d6-68f6-4194-a998-6ed09241b1fb.png">
s

ergunirem commented:
Since Olga approved, I am merging it like this and we can always change some styles.
